src/appleseed.python/bindproject.cpp

```diff
@@ -305,7 +305,9 @@ void bind_project()
     bpy::enum_<ProjectFileReader::Options>("ProjectFileReaderOptions")
         .value("Defaults", ProjectFileReader::Defaults)
         .value("OmitReadingMeshFiles", ProjectFileReader::OmitReadingMeshFiles)
-        .value("OmitProjectFileUpdate", ProjectFileReader::OmitProjectFileUpdate);
+        .value("OmitProjectFileUpdate", ProjectFileReader::OmitProjectFileUpdate)
+        .value("OmitSearchPaths", ProjectFileReader::OmitSearchPaths)
```

Comment:
I think that OmitSearchPaths is more accurate. It describes better what this option does.
Not only it does not add searchpaths from the project file, it also doesn't set the root path in the project.